scripts/check-format

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# Checks local /node_modules,
+# if not found will download that package and execute it.
+# Does not modify files, informs bad code format
+npx eslint .
```

Comment:
Until we've linted the codebase, we should keep this commented out.

Reply:
Eventually we could keep it like this and disable the CI in the .github/workflows folder, so it doesn't fail on every push

Reply:
Yeah that's fine too.

Reply:
[We can disable the workflow from the GitHub UI until everything is linted.](https://docs.github.com/es/actions/managing-workflow-runs/disabling-and-enabling-a-workflow#disabling-and-enabling-workflows-with-the-github-ui)

What do you think?